### PR TITLE
Add talk video

### LINKED ISSUE #91: Add video link in session detail
- add a new attribute in session model
- value each one from https://www.infoq.com/fr/mix-it
- render it in session detail
- profit, and make @sdeleuze happy (even if it's not OUR videos :p)

### PR DESCRIPTION
**Pull request by clacote**
Fix #91.

- [x] enrich model with `Session.video`.
- [x] render in detail.
- [x] value with each one of the ~70 video on InfoQ 😅 

This is at the moment only in session detail:
image
But maybe you would want to add it also in session lists, as it consumes not much screen estate. Up to you.

### CODE REVIEW

**Comment by clacote** [comment]
When adding an atribute in model, you have 3 DTO/entity classes to enrich: that's a lot for a schema-less like Mongo..

File: src/main/resources/templates/session.mustache [hunk 1]
```diff
@@ -10,7 +10,12 @@
         <div class="callout">
             <article>
                 <h2>{{session.title}}</h2>
-                <p class="subheader">{{session.event}} / #{{session.format}}</p>
+                <p class="subheader">
+                    {{session.event}} / #{{session.format}}
+                    {{#session.video}}
+                        / &#x25ba;&nbsp;Watch <a href="{{session.video}}">the video</a>
```
Comment (clacote) [inline src/main/resources/templates/session.mustache]: As Foundation seems to not come with a bultin icon library, I used a simple HTML entity. That's not even ugly:
image

File: src/main/resources/data/session/session_mixit14.json [hunk 2]
```diff
@@ -1107,6 +1119,7 @@
...
+    "video": "https://www.infoq.com/fr/presentations/spring4-dartangular",
```
Comment (clacote) [inline src/main/resources/data/session/session_mixit14.json]: Quel connard.

**Comment by clacote** [comment]
This does not build anymore with the recurring `Unresolved reference: Routes` 😓.
But this is done.

File: src/main/resources/data/session/session_mixit14.json (hunk 2 above)
Comment (sdeleuze) [inline src/main/resources/data/session/session_mixit14.json]: Ah ah

**Comment by sdeleuze** [comment]
Thanks !!!

File: src/main/resources/data/session/session_mixit14.json (hunk 2 above)
Comment (sdeleuze) [inline src/main/resources/data/session/session_mixit14.json]: Ils sont en train de faire un revival de Dart en ce moment d'ailleurs avec leur nouvel OS fushia (ou un nom dans le style)

**Pull request merged**